controllers/userController.ts

```diff
@@ -1,6 +1,6 @@
 import axios from "axios";
 import { User as UserProfile } from "../types/user";
-import { Facility as BathroomProfile } from "../types/facility";
+// import { Facility as BathroomProfile } from "../types/facility";
```

Comment:
Why is this commented out?

Reply:
oops let me remove all comments-- thanks for catching these